src/otx/algo/classification/mobilenet_v3_large.py

```diff
@@ -83,11 +96,21 @@ def __init__(
         )
 
     @property
-    def _export_parameters(self) -> dict[str, Any]:
-        """Defines parameters required to export a particular model implementation."""
-        parent_parameters = super()._export_parameters
-        parent_parameters.update({"via_onnx": True})
-        return parent_parameters
+    def _exporter(self) -> OTXModelExporter:
+        """Creates OTXModelExporter object that can export the model."""
+        mean, std = get_mean_std_from_data_processing(self.config)
+        return OTXNativeModelExporter(
+            task_level_export_parameters=self._export_parameters,
+            input_size=self.image_size,
+            mean=mean,
+            std=std,
+            resize_mode="standard",
+            pad_value=0,
+            swap_rgb=False,
+            via_onnx=True,  # NOTE: This should be done via onnx
+            onnx_export_configuration=None,
+            output_names=["logits", "feature_vector", "saliency_map"] if self.explain_mode else None,
```

Comment:
For `OTXNativeModelExporter` here and further it should be:
`output_names=["logits", "feature_vector", "saliency_map"] if self.explain_mode else ["logits"],`

Reply:
None also works, but for classification `logits` is preferable

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/3289/commits/63f5c60423bcf2abc90f9ec0807a593c325a497a

Reply:
Revert 
```
output_names=["logits", "feature_vector", "saliency_map"] if self.explain_mode else None
=>
output_names=["logits", "feature_vector", "saliency_map"] if self.explain_mode else ["logits"]
```
because many errors happened
https://github.com/openvinotoolkit/training_extensions/actions/runs/8641394317/job/23690872757#step:5:4447